swagger.yml

```diff
@@ -2010,6 +2010,36 @@ paths:
         name: path
         required: true
         type: string
+
+    put:
+      tags:
+        - objects
+      operationId: stageObject
+      summary: stage an object's metadata for the given branch
+      parameters:
+        - in: body
+          name: stats
+          schema:
+            $ref: "#/definitions/object_stats"
```

Comment:
This is too broad an object: it includes `mtime` and `path_type`, which AFAICT will (and should!) be ignored.  And it can keep on growing, meaning that all clients will need to keep on growing.